custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau.json

```diff
@@ -1200,19 +1191,16 @@
             {
               "type": "boolean_expression",
               "operator": "eq",
-              "property_value": "yes",
+              "property_value": "1",
```

Comment:
Realized the app was using 1s and 0s instead of yes for this particular question.